src/ATen/native/xpu/ReduceOps.cpp

```diff
@@ -485,7 +485,7 @@ Tensor XPUNativeFunctions::any(const Tensor& self) {
   return out;
 }
 
-// aten::any.any_out
+// aten::any.all_out
```

Comment:
Do we need this change ?

Reply:
We have no op named any.any, its actually any.all.